### DISCUSSION TITLE
Licenses question

### DISCUSSION THREAD

**Discussion by tuitui-x** [termux/termux-app#2760] [General]
In the LICENSE.md:

> Terminal Emulator for Android code is used which is released under Apache 2.0 license. Check terminal-view and terminal-emulator libraries.

@agnostic-apollo  Does it mean the libraries terminal-view and terminal-emulator are released under APL license? If not, which files of these libraries are released under APL?

**Comment by agnostic-apollo** [comment]
Yes, the entire libs are apache licensed. They were forked from https://github.com/jackpal/Android-Terminal-Emulator
